config/galaxy.ini.sample

```diff
@@ -324,12 +286,6 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # separated list.
 #tour_config_dir = config/plugins/tours
 
-# Webhooks directory: where to store webhooks - plugins to extend the Galaxy UI.
-# By default none will be loaded.  Set to config/plugins/webhooks/demo to load Galaxy's
-# demo webhooks.  To use an absolute path begin the path with '/'.  This is a comma
-# separated list.
-# webhooks_dir = config/plugins/webhooks
-
```

Comment:
Yes, we merged with the dev but somehow this file could not get merged. Now got it merged.